src/components/CouponDetails/index.jsx

```diff
@@ -634,8 +408,13 @@ class CouponDetails extends React.Component {
       assignments_remaining: `${code.redemptions.total - code.redemptions.used - code.redemptions.num_assignments}`,
       assignment_date: `${code.assignment_date}`,
       last_reminder_date: `${code.last_reminder_date}`,
-      is_public: code.is_public ? 'Public' : 'Private',
-      actions: this.getActionButton(code),
+      actions: <ActionButton
```

Comment:
@taliaedX I found the ActionButton usage!

Reply:
Ah hah! I don't know how I missed that one.